src/wazuh_db/wdb_upgrade.c

```diff
@@ -88,6 +88,7 @@ wdb_t * wdb_upgrade_global(wdb_t *wdb) {
 
     char db_version[OS_SIZE_256 + 2];
     int version = 0;
+    bool post_upgrade = FALSE;
```

Comment:
This variable prevents the post-upgrade snapshot creation if the pre-upgrade snapshot fails.
Maybe, we could try it regardless the previous result.

Reply:
Changed!